mmeds/summary.py

```diff
@@ -36,13 +39,14 @@ def summarize_qiime(summary_path, tool):
     # Get the mapping file
     copy(files['mapping'], files['summary'])
     copy(files['metadata'], files['summary'] / 'metadata.tsv')
-    copy(path / 'config_file.txt', files['summary'] / 'config_file.txt')
+    print('WHy is not this being usedd!!!?????')
+    copy(path / 'config_file.yaml', files['summary'] / 'config_file.yaml')
 
     # Get the template
     copy(STORAGE_DIR / 'revtex.tplx', files['summary'])
 
     # Load the configuration
-    config = load_config((path / 'config_file.txt').read_text(), files['metadata'], True)
+    config = load_config((path / 'config_file.yaml').read_text(), files['metadata'], True)
```

Comment:
I'm curious why you're calling read_text() on the config file rather than just passing it around as a dictionary.